### DISCUSSION TITLE
Commitments & Hybrid Benefit reports missing from 0.7 CostSummary.pbix?

### DISCUSSION THREAD

**Discussion by lawlormt** [microsoft/finops-toolkit#1176] [Q&A] [answered]
Hi Guys
Just having a look at the latest 0.7 release and in particular interested in the latest version of CostSummary powerBI report to check out some of the fixes.  
Have the "Commitments" & "Hybrid Benefit" reports been dropped from CostSummary? I was looking to check out this issue which is supposed to be addressed in this release: https://github.com/microsoft/finops-toolkit/issues/615
I'm using the CostSummary.storage.pbit from PowerBI-storage.zip.
As an aside the documentation download link here for "CostSummary" report now gives a 404.
https://microsoft.github.io/finops-toolkit/power-bi/cost-summary

Thanks
Michael

**Comment by flanakin** [comment, marked as answer]
TL;DR: Use the Rate optimization report.

---

These pages were duplicated in Cost summary and Rate optimization. When we added KQL reports for improved performance, we had to create a copy of all reports. This meant these pages were copied in 4 places, which cause a bit of extra manual processing that led to issues when they were out of sync. We removed the duplication to streamline our workflow.

As a side note, we gave discussed merging all reports into one. The Power BI guidance seems to suggest this is not recommended however. I am cusious what people prefer.
